Issues compiling v6.2.5 on Raspberry Pi 3B+

Hello,

I have a little pi cluster running v6.2.4 which compiles just fine. I tried to compile v6.2.5 but I get an error and I have no idea what I can do to fix it. Any good advice?

```
gcc -c -std=gnu99 -O2 -pipe -Iinclude/ -IOpenCL/ -Ideps/LZMA-SDK/C -Ideps/zlib -Ideps/zlib/contrib -Ideps/OpenCL-Headers -DWITH_BRAIN -Ideps/xxHash -DWITH_CUBIN -Ideps/unrar -DWITH_HWMON -D_7ZIP_ST -Wno-misleading-indentation 
deps/LZMA-SDK/C/7zCrc.c -o obj/7zCrc.LZMA.NATIVE.o -fpic
deps/LZMA-SDK/C/7zCrc.c: In function ‘CrcUpdateT0_32’:
deps/LZMA-SDK/C/7zCrc.c:167:9: warning: implicit declaration of function ‘__crc32b’ [-Wimplicit-function-declaration]
     v = __crc32b(v, *p++);
         ^~~~~~~~
deps/LZMA-SDK/C/7zCrc.c:176:11: warning: implicit declaration of function ‘__crc32w’ [-Wimplicit-function-declaration]
       v = __crc32w(v, *(const UInt32 *)(const void *)(p));
           ^~~~~~~~
deps/LZMA-SDK/C/7zCrc.c: In function ‘CrcUpdateT0_64’:
deps/LZMA-SDK/C/7zCrc.c:208:11: warning: implicit declaration of function ‘__crc32d’ [-Wimplicit-function-declaration]
       v = __crc32d(v, *(const UInt64 *)(const void *)(p));
           ^~~~~~~~
```

Thanks in advance.

--- a/docs/changes.txt
+++ b/docs/changes.txt
@@ -22,6 +22,7 @@
 ## Bugs
 ##
 
+- Fixed building error on Raspberry Pi
 - Fixed false negative on Unit Test in case of out-of-memory with grep in single mode
 - Fixed false negative on Unit Test with hash-type 25400
 - Fixed functional error when nonce-error-corrections that were set on the command line in hash-mode 22000/22001 were not accepted

--- a/src/Makefile
+++ b/src/Makefile
@@ -254,6 +254,14 @@ endif
 endif
 endif
 
+ifeq ($(UNAME),Linux)
+IS_RPI                  := $(shell grep -q Raspberry /proc/cpuinfo && echo 1 || echo 0)
+
+ifeq ($(IS_RPI),1)
+CFLAGS                  += -mcpu=cortex-a72
+endif
+endif
+
 CFLAGS                  += -pipe -Iinclude/ -IOpenCL/
 
 # LZMA

